src/vs/workbench/api/browser/viewsExtensionPoint.ts

```diff
@@ -501,7 +501,8 @@ class ViewsExtensionHandler implements IWorkbenchContribution {
 						originalContainerId: entry.key,
 						group: item.group,
 						remoteAuthority: item.remoteName || (<any>item).remoteAuthority, // TODO@roblou - delete after remote extensions are updated
-						hideByDefault: initialVisibility === InitialVisibility.Hidden
+						hideByDefault: initialVisibility === InitialVisibility.Hidden,
+						workspace: viewContainer?.id === REMOTE ? true : undefined
```

Comment:
As far as I can tell from the interface workspace is a boolean so is the ternary required?  Could we simplyfy to:
```
workspace: viewContainer?.id === REMOTE
```